src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -2374,7 +2380,10 @@ export function logScalePossibleForProfile(profileId: string) {
 
 export function logScalePossible(axisSelection: AxisMenuSelection) {
     if (axisSelection.dataType !== CLIN_ATTR_DATA_TYPE) {
-        if (axisSelection.dataType && axisSelection.isGenericAssayType) {
+        if (
+            isGenericAssaySelected(axisSelection) &&
```

Comment:
why do we bother checking above for CLIN_ATTR_DATA_TYPE?    we don't want to make log scale availabe for any of this?

Reply:
maybe put a a comment explaining this rule